packages/react-icons/scripts/build.js

```diff
@@ -180,8 +293,27 @@ async function writeIconModule(icon) {
     const entryDts = generateIconsEntry(icon.id, "dts");
     await appendFile(path.resolve(DIST, "all.d.ts"), entryDts, "utf8");
 
+    let count = 0;
```

Comment:
I might be missing some context here but it looks like count is redundant

```suggestion
```

Reply:
Yes, I've accidentally left that there. Removed in 5a955e2.